server/src/modules/answers/answers.controller.ts

```diff
@@ -68,35 +59,16 @@ export class AnswersController {
       if (!hasAnswerPermissions) {
         return res
           .status(403)
-          .json(
-            helperFunction.responseHandler(
-              false,
-              403,
-              'You do not have permissions to answer question',
-              null,
-            ),
-          )
+          .json({ message: 'You do not have permissions to answer question' })
       }
       // Save Answer in the database
-      const [error, data] = await this.answersService.createAnswer({
+      const data = await this.answersService.createAnswer({
         body: req.body.text,
         userId: req.user.id,
         postId: req.params.id,
       })
 
-      if (error) {
-        logger.error({
-          message: 'Error while adding new answer',
-          meta: {
-            function: 'addAnswer',
-            userId: req.user.id,
-            postId: req.params.id,
-          },
-          error,
-        })
-        return res.status(error.code).json(error)
-      }
-      return res.status(data?.code || 200).json(data)
+      return res.status(201).json(data)
```

Comment:
since the scope of the PR is very wide, avoid taking the opportunity to change biz logic